lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -72,6 +72,56 @@ CrossOp::inferReturnTypes(MLIRContext *context, std::optional<Location> loc,
   return success();
 }
 
+LogicalResult FilterOp::verifyRegions() {
+  MLIRContext *context = getContext();
+  Type si1 = IntegerType::get(context, /*width=*/1, IntegerType::Signed);
+  Region &condition = getCondition();
+
+  // Verify that type of yielded value is Boolean.
+  auto yieldOp = llvm::cast<YieldOp>(condition.front().getTerminator());
```

Comment:
mlir::cast ? (its same thing/alias but just more consistent with other usage)

Reply:
I wasn't aware of the alias! But I don't have the feeling that is being used consistently... `llvm::cast` shows 1249 hits in upstream MLIR, `mlir::cast` shows 10 hits. (That doesn't account for `using mlir` but I guess it gives *some* indication...)

If you think we should prefer `mlir::cast`, ping me and I'll address it for the whole repo. (We have been using `llvm::cast` or whichever thing `cast` was resolved for everywhere.)